config/systems-uenv/clariden-uenv.py

```diff
@@ -128,17 +128,28 @@
         'target_systems': ['clariden']
     }
     env.update(v)
-    activation_script = v['activation']
+    activation = v['activation']
+
+    # FIXME: Assume that an activation script is given, to be sourced
+    if isinstance(activation, str):
+        if not activation.startswith(image_mount):
+            raise ConfigError(
+                    f'activation script of {k!r} is not consistent '
+                    f'with the mount point: {image_mount!r}')
+
+        env['prepare_cmds'] = [f'source {activation}']
+    elif isinstance(activation, list):
+        env['prepare_cmds'] = activation
+    else:
+       raise ConfigError(
+           f'activation has to be either a string or list of strings'
```

Comment:
Can we change the message to `activation should be a list of commands that configure the environment when run`.
We want to deprecate the single string as an activation script, so let's not mention it in the error message?

Reply:
I will remove once we have updated the images.